key.js

```diff
@@ -202,76 +479,41 @@ const entireLogic = () => {
         }
     })
 }
-const levels = () => {
-    entireLogic();
-}
+
 // Timer Style :
 timerElement.style.fontSize = `45px`
 timerElement.style.display = `none`;
-h3.style.display = `none`
-let bool = false;
-
+sequenceDisplay.style.display = `none`
 keyBoard.style.display = `none`;
-startButton.style.transform = `translateY(300px)`;
 const levelOne = () => {
     lives++;
-    h3.display = `none`;
+    sequenceDisplay.display = `none`;
     keyBoard.style.display = `none`;
     startButton.style.transform = `translateY(200px)`;
     startButton.style.fontSize = `100px`;
 }
 
-const homeAnimation = () => {
-    clearInterval(timer);
-    timerElement.style.display = `none`;
-    headings.innerText = `KEY FOCUS`
-    headings.style.color = ``;
-    startButton.style.fontSize = `100px`
-    startButton.textContent = `START`;
-    document.body.style.backgroundColor = ``;
-    h3.style.display = `none`;
-    startButton.style.transform = `translateY(300px)`
-    keyBoard.style.display = `none`;
-    startButton.style.color = ``;
-    startButton.style.backgroundColor = ``;
-    startButton.style.border = ``;
-    span.forEach(element => {
-        element.innerText = ``;
-    })
-}
-
-const startAnimation = () => {
-    timerElement.style.display = ``;
-    newSequence();
-    startButton.style.fontSize = `45px`
-    let seconds = 60;
-    timer = setInterval(() => {
-        seconds--;
-        timerElement.textContent = `${seconds} `;
-        if (!seconds) {
-            clearInterval(timer);
-        }
-    }, 1000)
-    timerElement.style.boxShadow = `0px 0px 4px 3px darkgray`;
-    startButton.style.transform = ``
-    h3.style.display = ``;
-    keyBoard.style.display = ``;
-    startButton.textContent = `HOME`;
-    startButton.style.transform = `translateX(600px)`
-    startButton.style.backgroundColor = `red`;
-    startButton.style.color = `white`;
-    document.body.style.backgroundColor = `whitesmoke`;
-    startButton.style.border = `5px solid darkred`
-    headings.textContent = `Baby Level`;
-    headings.style.color = "darkslategray";
-    levels();
-}
 startButton.addEventListener('click', () => {
     buttonPress()
     bool = !bool;
+    console.log(bool);
     if (bool) {
         startAnimation();
     } else {
         homeAnimation();
     }
 })
+
+function logic() {
+    // ...
+    // ...
+    gameModule.incrementGameLevel();
+}
+
+// gameModule
+// gameController
+
+
+const controller = (function (game, view) {
+    startButton.addEventListener('click', game.gameState.startGame());
+})(gameModule, viewModule)
```

Comment:
Add an extra line after any javascript / html / css file.  Please read [this](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file) for more information.

```suggestion
})(gameModule, viewModule)

```